src/pudl/output/ferc1.py

```diff
@@ -1747,6 +1749,18 @@ def calc_idx(self: Self) -> list[str]:
         """Primary key columns for calculations in this explosion."""
         return [col for col in list(NodeId._fields) if col in self.exploded_pks]
 
+    def prep_table_to_explode(
+        self: Self, table_name: str, table_df: pd.DataFrame
+    ) -> pd.DataFrame:
+        """Prep an core input table for explosion."""
```

Comment:
Can you make this slightly more descriptive? "Assign table name and rename factoid column in preparation for explosion" or something similar?